client/blocks/comments/comment-count.jsx

```diff
@@ -15,22 +14,28 @@ import { localize } from 'i18n-calypso';
 const CommentCount = ( { count, translate } ) => {
 	let countPhrase;
 	if ( count > 0 ) {
-		countPhrase = translate( '%(commentCount)d comment', '%(commentCount)d comments', {
-			count,
-			args: {
-				commentCount: count,
+		countPhrase = translate(
+			'{{span}}%(commentCount)d comment{{/span}}',
```

Comment:
Hi! I've found a possible matching string that has already been translated **41** times:
`translate( '{{count/}}{{span}}Comment{{/span}}', '{{count/}}{{span}}Comments{{/span}}' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).